docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,114 @@
+# Release 393 (10 August 2022)
+
+## General
+
+* Add support for the [SQL `Merge` statement](/sql/merge). ({issue}`7933`)
+
+## Security
+
+* Fix potential incorrect rejection of tokens when OAuth 2.0 refresh-tokens are
+  not obtained from IdP but were expected. ({issue}`13168`)
+* Fix showing outdated OAuth 2.0 refresh tokens in the web UI. ({issue}`13168`)
+
+## Docker image
+
+* Add image for `ppc64le`. ({issue}`13522`)
+
+## ClickHouse connector
+
+* Add support for the `unsupported-type-handling` catalog configuration
```

Comment:
Was the docs added (its just an include I think)

Reply:
The docs were already there! Which is a little alarming.